todo-app/src/App.js

```diff
@@ -1,10 +1,86 @@
 import React, { Component } from 'react';
+import Header from './Components/Header'
+import Footer from './Components/Footer'
+import Container from './Components/Container'
+
+
+import './App.css';
+const data = require('./todoList.json');
 
 class App extends Component {
+  state = {
+    todoItems: []
+  }
+  componentWillMount() {
+    localStorage.getItem('todoItems') && this.setState({
+      todoItems: JSON.parse(localStorage.getItem('todoItems'))
+    })
+  }
+  componentDidMount() {
+    if (!localStorage.getItem('todoItems')) {
+      this.setState({
+        todoItems: data
+      })
+    } else {
+      console.log('we are using the data from the Local Storage.')
+    }
+
+  }
+  componentWillUpdate(nextProps, nextState) {
+    localStorage.setItem('todoItems', JSON.stringify(nextState.todoItems))
+  }
+
+  handelSelected = (index) => {
```

Comment:
Using index to find an item in an array is a pretty fragile method. Please use an identifier like ID to do this.